language-extensions/dotnet-core-CSharp/sample/regex/README.md

```diff
@@ -151,9 +152,7 @@ You need to create an external language in the database. The external language i
 
 If you are using Windows, follow the steps below to create an external language for .NET.
 
-1. Create a .zip file containing the extension.
-
-    Follow this [tutorial]((../../../../README.md)) to create **dotnet-core-CSharp-lang-extension.zip**. This file contains the **nativecsharpextension.dll** and **Microsoft.SqlServer.CSharpExtension.dll**.
+1. Download the [release](https://github.com/microsoft/sql-server-language-extensions/releases/download/dotnet-csharp-v1.0.0/dotnet-core-CSharp-lang-extension-windows-release.zip) containing the language extension. If you have any changes to the language extension, then follow this [tutorial](../../README.md) to create **dotnet-core-CSharp-lang-extension.zip**. This file contains the **nativecsharpextension.dll** and **Microsoft.SqlServer.CSharpExtension.dll**.
```

Comment:
>/dotnet-csharp-v1.0.0 [](http://example.com/codeflow?start=103&length=21)

I don't think we should link to specific release version. Can we link to generic release page?

Reply:
Yes, I changed it to link to the generic release page. Thanks.